sdk/eventgrid/azure-eventgrid/tests/recordings/test_eg_publisher_client.test_send_cloud_event_data_with_extensions.yaml

```diff
@@ -0,0 +1,40 @@
+interactions:
+- request:
+    body: '[{"id": "9883be2c-2252-4c42-bb32-31317b72237f", "source": "http://samplesource.dev",
+      "data": "cloudevent", "type": "Sample.Cloud.Event", "time": "2020-09-03T08:23:07.846089Z",
+      "specversion": "1.0"}]'
+    headers:
+      Accept:
+      - '*/*'
+      Accept-Encoding:
+      - gzip, deflate
+      Connection:
+      - keep-alive
+      Content-Length:
+      - '198'
+      Content-Type:
+      - application/cloudevents-batch+json; charset=utf-8
+      User-Agent:
+      - azsdk-python-eventgridpublisherclient/unknown Python/3.7.3 (Windows-10-10.0.18362-SP0)
+      aeg-sas-key:
```

Comment:
We shall declare this header in the recording black list, this is secret exposed